application/common/models/User.php

```diff
@@ -292,18 +282,9 @@ public function fields(): array
             'locked',
         ];
 
-        if ($this->hasPasswordAlready()) {
-            $fields['password_expires_at_utc'] = function () {
-                return $this->getPasswordExpiration();
-            };
-
-            $fields['password_last_changed'] = function () {
-                /** @var $mostRecentPassword PasswordHistory */
-                $mostRecentPassword = $this->getPasswordHistories()
-                                           ->orderBy(['id' => SORT_DESC])
-                                           ->one();
-
-                return $mostRecentPassword->created_utc;
+        if ($this->password_id !== null) {
```

Comment:
I saw mixed uses, some calls to the method and some simply checking for `null`.  I hope with the new column name, this may not be as big an issue. (https://github.com/silinternational/idp-id-broker/commit/c9752a3c1979dea7158a1eb89d0b6ca8742343c5)